### PR TITLE
Move formatter config to configuration file

### PR DESCRIPTION
**Pull request by kellertuer**
This also makes running formatter locally easier because you don't have to pass the style anymore.

If this is ok, I would also apply this change to our other packages (just doing one commit to master).

### CODE REVIEW

**Comment by kellertuer** [comment]
We could actually also shorten this to just have

````
style = blue
````

see https://domluna.github.io/JuliaFormatter.jl/dev/blue_style/
what do you think?

**Comment by mateuszbaran** [comment]
It looks reasonable. That `style = blue` seems like a work in progress right now (new blue style rules are still being added) so maybe we could hold off on that to not disrupt PRs to Manifolds.jl with style tweaks.

**Comment by kellertuer** [comment]
Yes, it seems to be still work in progress, I enabled it that way n Manopt just to try, but the reasonable one is the one currently in this PR.

**Review by mateuszbaran — APPROVED** [review]
